### PR TITLE
Feature/setup routes gallery app

### PR DESCRIPTION
**Pull request by Keith-Go**
## Ticket
- Setup Route Gallery App

### CODE REVIEW

File: gallery_app/lib/configs/routes/routes_config.dart [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+import 'package:gallery_app/modules/row_column_expanded/screens/row_column_expanded_screen.dart';
+import 'package:gallery_app/modules/scaffold_bottomappbar_actionfloatbutton/screens/scaffold_bottomappbar_actionfloatbutton_screen.dart';
+import 'package:gallery_app/modules/stack_align/screens/stack_align_screen.dart';
+import 'package:gallery_app/modules/text_richtext_widget-span/screens/text_richtext_widget_span_screen.dart';
+import 'package:gallery_app/modules/textfield/screens/textfield_screen.dart';
+import 'package:gallery_app/modules/wrap_chip/screens/wrap_chip_screen.dart';
+
+Route<dynamic>? onGenerateRoute(RouteSettings setting) {
+  switch (setting.name) {
+    case "/":
```
Comment (luke-nguyen-goldenowl) [inline gallery_app/lib/configs/routes/routes_config.dart]: Create a RouteNames class to contain all the route names of the screens. When used, call from that class to use. When using any route name, it is taken from that class

File: gallery_app/lib/modules/home/model/data_home.dart [hunk 2]
```diff
@@ -1,66 +1,87 @@
...
+  DataHome(
+      {required this.icon,
+      required this.title,
+      required this.describe,
+      required this.routeName});
 }
 
 List<DataHome> listData = [
   DataHome(
+    routeName: "/cupertino",
```
Comment (luke-nguyen-goldenowl) [inline gallery_app/lib/modules/home/model/data_home.dart]: Taken from the RouteNames class above

File: gallery_app/lib/modules/home/model/data_home.dart [hunk 3]
```diff
@@ -1,66 +1,87 @@
...
     title: "Cupertino",
-    icon: Icons.text_fields,
+    icon: Icons.phone_android,
     describe:
         'Beautiful and high-fidelity widget for current iOS design of the predefined text styles',
   ),
   DataHome(
+    routeName: "/text_richtext_span",
```
Comment (luke-nguyen-goldenowl) [inline gallery_app/lib/modules/home/model/data_home.dart]: Same for all route name data

**Comment by Keith-Go** [comment]
Anh check lại giúp em nha

**Comment by luke-nguyen-goldenowl** [comment]
Approved

**Pull request merged**